about.js

```diff
@@ -0,0 +1,13 @@
+function grab(e) {
+  return document.querySelector(`.${e}`);
+}
+
```

Comment:
- [ ] Please change this to es6 arrow function syntax as requested in the JavaScript best practices.

Reply:
fixed